components/app/AppDisplayContact.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="show-contact grid grid-cols-2 gap-4">
+    <div class="border-t border-b border-gray-400 p-5">
+      <span>
+        {{ name }}
+      </span>, {{ orgName }}
+      <br />
+      {{ email }}
+    </div>
+    <div v-if="type != ''" class="border-t border-b border-gray-400 p-5">
```

Comment:
I believe in vue v-if="type !=''" is the same as just v-if="type"

Reply:
Also why are you using a new component when there is already one ?

Reply:
ok, it works